### PR TITLE
S3 documentation revamp

### PR DESCRIPTION
**Pull request by ameypv-rudder**
## Description of the change

> Updated S3 documentation to fix the following aspects:
 - Readability
 - Standardize URLs, internal links, and image paths
 - Add section on using S3 as an intermediary object storage in case of warehouse destinations

## Type of documentation

- [ ] New documentation
- [x] Documentation update
- [ ] Hotfix

## Notion ticket link

> [Warehouse permissions](https://www.notion.so/rudderstacks/Warehouse-Destinations-Update-object-storage-permissions-6f134583b3f942f1a9c829f6af3ae501)

### CODE REVIEW

**Review by achettyiitr — APPROVED** [review]
LGTM

File: docs/destinations/storage-platforms/amazon-s3.mdx [hunk 1]
```diff
@@ -104,67 +108,93 @@ If you are hosting RudderStack on your instances and don't want to follow option
...
-<img src="../../assets/screenshot-2020-06-01-at-2.29.13-pm.png" />
-<span class="imageTitle">S3 Destination Settings</span>
+In the **Connection Settings** page, enter the following settings to configure the S3 destination: 
 
-<div class="infoBlock">
+<img src="../../assets/event-stream-destinations/s3-connection-settings.png" alt="S3 connection settings" />
 
-If the AWS credentials are already configured on your data plane's environment via [environment credentials](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials) or by following the [below steps](https://rudderstack.com/docs/destinations/storage-platforms/amazon-s3/#permissions), you can skip adding the credentials in this step. Only the S3 Bucket Name is required.
+- **S3 Bucket Name**: Enter your S3 bucket name.
+- **Prefix**:
```
Comment (Jayachand) [inline docs/destinations/storage-platforms/amazon-s3.mdx:151]: prefix content is empty

**Review by abhimanyubabbar — APPROVED** [review]
LGTM 🎈

**Review by Jayachand — APPROVED** [review]

File: docs/destinations/storage-platforms/amazon-s3.mdx [hunk 2]
```diff
@@ -1,34 +1,35 @@
...
 
-- Login to your [Amazon AWS S3 console](https://aws.amazon.com/console/).
-- Create a new bucket. Alternatively, you can also choose already existing bucket.
+1. Login to your [Amazon AWS S3 console](https://aws.amazon.com/console/).
+2. Create a new bucket. Alternatively, you can also choose an already-existing bucket.
 
 ## Permissions
 
-The Following are two ways to give permissions to RudderStack for writing in your bucket. You can choose one of them based on your internal security policies. The First option is the easiest.
+There are two ways to give RudderStack the relevant permissions for writing in your bucket. You can choose any one of them based on your internal security policies.
```
Comment (ishachopra0401) [inline docs/destinations/storage-platforms/amazon-s3.mdx:23]: we are mentioning 3 options below, can modify this

File: docs/destinations/storage-platforms/amazon-s3.mdx [hunk 3]
```diff
@@ -104,67 +108,93 @@ If you are hosting RudderStack on your instances and don't want to follow option
...
 ```markup
-RUDDER_AWS_S3_COPY_USER_ACCESS_KEY_ID=<above created user access key>
-RUDDER_AWS_S3_COPY_USER_ACCESS_KEY=<above created user secret access key>
+RUDDER_AWS_S3_COPY_USER_ACCESS_KEY_ID=<access_key_id>
+RUDDER_AWS_S3_COPY_USER_ACCESS_KEY=<secret_access_key>
+```
+
+### S3 permissions for warehouse destinations
+
+If you're using your S3 bucket as an **intermediary object storage** for the <Link to="/data-warehouse-integrations/">warehouse destination</Link>, then you will need the following bucket permissions:
```
Comment (ishachopra0401) [inline docs/destinations/storage-platforms/amazon-s3.mdx:126]:
```suggestion
If you're using your S3 bucket as an **intermediary object storage** for a <Link to="/data-warehouse-integrations/">warehouse destination</Link>, you will need the additional bucket permissions as below:
```

File: docs/destinations/storage-platforms/amazon-s3.mdx [hunk 4]
```diff
@@ -104,67 +108,93 @@ If you are hosting RudderStack on your instances and don't want to follow option
...
 ```
 
-## **Configuring** Amazon S3 **in RudderStack**
+## Configuring S3 destination in RudderStack
 
-In order to enable dumping data to Amazon S3, you will first need to add it as a destination to the source from which you are sending event data. Once the destination is enabled, events from RudderStack will start flowing to Amazon S3. You will start seeing the files created in your S3 bucket.
+Follow these steps to set up S3 as a destination in RudderStack:
 
-To do so, please follow these steps:
+1. From your [RudderStack dashboard](https://app.rudderstack.com/), add a source. Then, from the list of destinations, select **Attribution**.
```
Comment (ishachopra0401) [inline docs/destinations/storage-platforms/amazon-s3.mdx:141]:
```suggestion
1. From your [RudderStack dashboard](https://app.rudderstack.com/), add a source. Then, from the list of destinations, select **Amazon S3**.
```

File: docs/destinations/storage-platforms/amazon-s3.mdx (hunk 2 above)
Comment (ameypv-rudder) [inline docs/destinations/storage-platforms/amazon-s3.mdx:23]:
```suggestion
There are three ways to give RudderStack the relevant permissions for writing in your bucket. You can choose any one of them based on your internal security policies.
```

File: docs/destinations/storage-platforms/amazon-s3.mdx (hunk 3 above)
Comment (ameypv-rudder) [reply docs/destinations/storage-platforms/amazon-s3.mdx:126]: `PutObject` is a permission already included in the Permissions section above. Here we are listing **all** the permissions required to use S3 as an intermediary object storage for sending events to a WH destination.

**Review by ishachopra0401 — APPROVED** [review]

**Pull request merged**